Library/Formula/libffi.rb

```diff
@@ -21,15 +16,35 @@ class Libffi < Formula
     depends_on "libtool" => :build
   end
 
-  keg_only :provided_by_osx, "Some formulae require a newer version of libffi."
+  keg_only :provided_by_osx," Some formulae require this version of libffi."
+
+  stable do
+    patch :DATA
```

Comment:
Please add a comment above this line explaining why the patch is necessary as well as a link to the upstream’s bug report.